Makes rviz trajectory visualization topic relative (#2835)
ros-planning/moveit: MIGRATION.md

- The joint states of `passive` joints must be published in ROS and the CurrentStateMonitor will now wait for them as well. Their semantics dictate that they cannot be actively controlled, but they must be known to use the full robot state in collision checks. (https://github.com/ros-planning/moveit/pull/2663)
- Removed deprecated header `moveit/macros/deprecation.h`. Use `[[deprecated]]` instead.
- All uses of `MOVEIT_CLASS_FORWARD` et. al. must now be followed by a semicolon for consistency (and to get -pedantic builds to pass for the codebase).

> - The default topic for the Rviz trajectory visualization tool now uses the relative instead of the absolute namespace (i.e. `move_group/display_planned_path` instead of `/move_group/display_planned_path`).

Reviewer: ```suggestion
- In case you start RViz in a namespace, the default topic for the trajectory visualization display now uses the relative instead of the absolute namespace (i.e. `<ns>/move_group/display_planned_path` instead of `/move_group/display_planned_path`).
```

> - The Rviz trajectory visualization tool now looks for the trajectory topic to be on the relative ns instead of the absolute ns when it is first added to Rviz (i.e. `move_group/display_planned_path` vs. `/move_group/display_planned_path`).

Reviewer 2: ```suggestion
- The default topic for the Rviz trajectory visualization tool now uses the relative instead of the absolute namespace (i.e. `move_group/display_planned_path` instead of `/move_group/display_planned_path`).
```
